activesupport/lib/active_support/multibyte/unicode.rb

```diff
@@ -138,21 +138,6 @@ def pack_graphemes(unpacked)
         unpacked.flatten.pack("U*")
       end
 
-      # Re-order codepoints so the string becomes canonical.
-      def reorder_characters(codepoints)
```

Comment:
I noticed that tests will fail on Ruby 2.2 because supported Unicode version is different :cry: